docs/src/content/docs/book/learn-tact-in-y-minutes.mdx

```diff
@@ -0,0 +1,1484 @@
+    // with up to 4 reference to other cells. Cyclic references are prohibited.
+    let emptyC: Cell = emptyCell(); // well, cannot do much with that
+
+    // Builder is an immutable primitive to construct (compose) cells.
+    let bb: Builder = beginCell()
+        .storeUint(42, 6)  // storing 42 using 6 bits
+        .storeInt(42, 7)   // storing 42 using 7 bits (signed Int)
+        .storeBool(true)   // writing 1 as a single bit
+        .storeBit(true)    // alias to storeBool()
+        .storeCoins(40)    // common way of storing nanoToncoins
+        .storeAddress(myAddress())
+        .storeRef(emptyC); // storing a reference
+    let composed: Cell = bb.endCell();
+
+    // Slice is a mutable primitive to deconstruct (parse) cells.
+    let target: Slice = composed.asSlice(); // let's start parsing `composed` Cell
+
+
+    // The type ascription is optional for most cases except for maps
+    // and optional types, but we'll discuss those in the
+    // "Composite types" section below.
+    let fortyTwo = target.loadUint(6); // taking 6 bits out of the `target` Slice,
+                                       // mutating it in the process
+
+    // If you don't want the result, you can ignore it with a wildcard.
+    let _ = target.loadInt(7);
+
+    // Finally, there are methods to skip the value, i.e., to discard it.
+    target.skipBool();
+
+    // Manual composition and parsing of Cells is tedious,
+    // error-prone and is generally not recommended.
+    // Instead, prefer using structures: struct and message struct types.
+    // See the "Composite types" section below for more info.
+
+
+    // ---
+    // Strings are immutable sequences of characters,
+    // which are useful to store texts to be used as special message bodies.
+    // ---
+
+    // String literals are wrapped in double-quotes and can contain escape sequences,
+    // but they intentionally cannot be concatenated via any operators.
+    let str: String = "I am a string literal, boo!";
+    let escape1 = "\\ \" \n \r \t \v \b \f \x00 through \xFF";
+    let escape2 = "\u0000 through \uFFFF and \u{0} through \u{FFFFFF}";
+
+    // Strings are useful for storing text,
+    // so they can be converted to a Cell type to be used as message bodies.
+    let noComments: Cell = "yes comments".asComment(); // prefixes a string with 32 zero bits
+
+
+    // ---
+    // It's cheaper to manipulate and prepare strings off-chain,
+    // but there are StringBuilders for composing them on-chain.
+    // ---
+
+    // The interfaces look similar to the Builders because, under the hood,
+    // StringBuilders are just Builders with a different set of functions exposed by Tact.
+    let sb: StringBuilder = beginString()
+        .concat("You can be The Good Guy or the guy who saves the world...")
+        .concat("You can't be both");
+
+    // You can append something to an existing StringBuilder too.
+    sb.append(". — Radcliffe");
+    let fullQuote: String = sb.toString();
```

Comment:
let's remove it, do not mention here something that is not needed for 90% of smart contracts